Documentation/operations/upgrade.rst

```diff
@@ -283,27 +283,44 @@ combination is not listed in the table below, then it may not be safe. In that
 case, consider performing incremental upgrades between versions (e.g. upgrade
 from ``1.10.x`` to ``1.11.y`` first, and to ``1.12.z`` only afterwards).
 
-+-----------------------+-----------------------+-------------------------+---------------------------+
-| Current version       | Target version        | L3/L4 impact            | L7 impact                 |
-+=======================+=======================+=========================+===========================+
-| ``1.12.x``            | ``1.13.y``            | Minimal to None         | Clients must reconnect[1] |
-+-----------------------+-----------------------+-------------------------+---------------------------+
-| ``1.11.x``            | ``1.12.y``            | Minimal to None         | Clients must reconnect[1] |
-+-----------------------+-----------------------+-------------------------+---------------------------+
-| ``1.10.x``            | ``1.11.y``            | Minimal to None         | Clients must reconnect[1] |
-+-----------------------+-----------------------+-------------------------+---------------------------+
++-----------------------+-----------------------+----------------------------+---------------------------+
+| Current version       | Target version        | L3/L4 impact               | L7 impact                 |
++=======================+=======================+============================+===========================+
+| ``1.13.4``            | ``1.13.y``            | Lingering Connections[2]   | Clients must reconnect[1] |
++-----------------------+-----------------------+----------------------------+---------------------------+
+| ``1.13.x``            | ``1.13.4``            | Interrupted Connections[3] | Clients must reconnect[1] |
++-----------------------+-----------------------+----------------------------+---------------------------+
+| ``1.12.x``            | ``1.13.y``            | Minimal to None            | Clients must reconnect[1] |
```

Comment:
Couple of notes:

- For 1.13.x (x <= 3) upgrade to 1.13.y (y >= 5), it should also be minimal to none.
- For 1.12.x to 1.13.y we should probably recommend y >= 5 

Given the complexity of enumerating these, and given that we'll have a version with minimal disruption coming soon, I'm not sure it is worth documenting in this specific table. It gets awfully complicated to list the combinations.

Currently for any combinations that have minimal to no impact, we list the versions. I think that implies there are no guarantee for other versions.

One counter-proposal here would be just to update the existing table to:
1. Add 1.13.x (x <= 3) to 1.13.y (y >= 5) has minimal to none disruption
2. Update the 1.12.x to 1.13.y to specify y >= 5.

Reply:
Related, we never assert support for older patch releases. If someone is running 1.13.0 today, it's clear that they should upgrade to the latest 1.13.y release in order to adopt all bugfixes. After newer patch releases are out, it's unusual for users to decide to upgrade to the N-1 patch release if the N patch release is out. From that perspective, I think it makes sense to (a) assume that users will attempt to upgrade to the latest patch release for the series and document well for that scenario, and (b) have a separate note that clearly defines what happens if they upgrade to the version that has the issues (1.13.4). The rest of the PR mostly covers (b) already. My feedback above on this table is trying to improve (a).

Reply:
I've added your feedback (I think). Can you verify this is what you wanted?